app/views/movies/_posters_carousal.html.erb

```diff
@@ -0,0 +1,30 @@
+ <!-- this div shows movie posters in a bootstrap carousal-->
+<div class = 'pt-5'>
+  <h3>Movie Posters</h3>
+  <div id="carouselExampleIndicators" class="carousel slide " data-ride="carousel">
+    <ol class="carousel-indicators">
+      <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active"></li>
+      <li data-target="#carouselExampleIndicators" data-slide-to="1"></li>
+      <li data-target="#carouselExampleIndicators" data-slide-to="2"></li>
+    </ol>
+    <div class="carousel-inner">
+      <% @movie.posters.each_with_index do |poster, index| %>
+        <% if index == 0 %>
+          <div class="carousel-item active">
+        <% else %>
+          <div class="carousel-item">
+        <% end %>
```

Comment:
Create a helper method like `carousel_class(index)` and use it in div like below:
`<div class="<%= carousel_class(index) %>">`
This method return the class name based on index value